### PR TITLE
feat: add getting started into README

### PR DESCRIPTION
**Pull request by hoangSTS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,14 +29,30 @@ You also need to install the dependencies required by React Native.
...
 Follow instructions for your given `development OS` and `target OS`.
 
-## Quick Start
+## Getting Started
+
+- Prerequisites
+
+  - Node.js and yarn should be installed on your system.
+  - Git should be installed on your system.
+  - [React Native CLI should be installed on your system.](https://reactnative.dev/docs/environment-setup)
```
Comment (loc-nguyenthien) [inline README.md:38]: These info are very identical to above `Requirement` section, I think we can remove it

**Review by loc-nguyenthien — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hoangSTS) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -29,14 +29,22 @@ You also need to install the dependencies required by React Native.
...
+  ```
+
+- Start the app
+  ```bash
+  yarn android
+  ```
+  ```bash
+  yarn ios
+  ```
+  [Check out the full documentation!](#documents)
```
Comment (loido) [inline README.md:47]: i think, we should check out full docs at folder docs 
https://github.com/saigon-technology/rn-base-project/tree/develop/docs

**Review by loido — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -110,6 +118,7 @@ Thanks goes to these wonderful people
 </table>
 
 ## Contributing
+
```
Comment (TranQuangPhi) [inline README.md:121]: please remove this line

**Review by loc-nguyenthien — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (hoangSTS) [reply README.md:47]: I think that moving to the Document below is better than open new tab.

File: README.md [hunk 4]
```diff
@@ -29,14 +29,22 @@ You also need to install the dependencies required by React Native.
...
-npx react-native init ProjectName --template rn-base-project
-```
-[Check out the full documentation!](#documents)
+  ```bash
+  npx react-native init YourProjectName --template rn-base-project
+  ```
+
+- Start the app
+  ```bash
+  yarn android
```
[the remark this answers is not in the document]
Comment (hoangSTS) [reply README.md:42]: removed.

File: README.md (hunk 3 above)
Comment (hoangSTS) [reply README.md:121]: removed.

**Review by TranQuangPhi — APPROVED** [review]

**Review by loido — APPROVED** [review]

**Pull request merged**